editor/editor_inspector.cpp

```diff
@@ -1939,7 +1940,7 @@ void EditorInspector::update_tree() {
 							ep->set_label(F->get().label);
 						} else {
 							//use existin one
-							ep->set_label(name);
+							ep->set_label(TTR(name));
```

Comment:
`TTR()` shouldn't be used with variable names as there will be no way for the PO file generator to pick up property names.

Reply:
I edit extract.py, can extract _initial_set